src/go/rdctl/cmd/shell.go

```diff
@@ -116,21 +117,34 @@ func setupLimaHome() error {
 const restartDirective = "Either run 'rdctl start' or start the Rancher Desktop application first"
 
 func checkLimaIsRunning(commandName string) bool {
-	// Ignore error messages; none are expected here
-	output, err := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}").CombinedOutput()
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+
+	cmd := exec.Command(commandName, "ls", "0", "--format", "{{.Status}}")
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Run()
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to run 'rdctl shell': %s\n", err)
+		fmt.Fprintf(os.Stderr, "Failed to run '%s ls 0 --format {{.Status}}': %s\n", commandName, err)
```

Comment:
Don't repeat (and hard-code) the command arguments. `cmd` has a `String()` method, so you can print it directly:

```suggestion
		fmt.Fprintf(os.Stderr, "Failed to run %q: %s\n", cmd, err)
```

Reply:
From the docs: at https://pkg.go.dev/builtin#string:

"String returns a human-readable description of c. It is intended only for debugging. In particular, it is not suitable for use as input to a shell. The output of String may vary across Go releases."

I haven't been able to get this message, and it's less code, so I'm ok with it.

Reply:
You can try it on the playground: https://go.dev/play/p/CtTl3hOizyf